misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -38,13 +38,12 @@ class JettyService @Inject internal constructor(
         val serverConnector = ServerConnector(server)
         serverConnector.port = webConfig.port
         serverConnector.idleTimeout = webConfig.idle_timeout
-        val servletHandler = ServletHandler()
-        servletHandler.addServletWithMapping(ServletHolder(webActionsServlet), "/*")
-
         server.addConnector(serverConnector)
-        server.addManaged(servletHandler)
 
-        server.handler = servletHandler
+        val context = ServletContextHandler(ServletContextHandler.SESSIONS)
```

Comment:
Are SESSIONS required for websockets? I think we strongly want to avoid enabling sessions.